pom.xml

```diff
@@ -62,19 +62,20 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>4.3.1</cdap.version>
+    <cdap.version>4.3.3</cdap.version>
     <guava.version>20.0</guava.version>
     <slf4j.version>1.7.5</slf4j.version>
     <junit.version>4.12</junit.version>
     <hadoop.version>2.8.0</hadoop.version>
     <hydrator.version>1.8.0</hydrator.version>
-    <bigquery.connector.hadoop2.version>0.10.2-hadoop2</bigquery.connector.hadoop2.version>
-    <google.cloud.bigquery.version>0.25.0-beta</google.cloud.bigquery.version>
+    <bigquery.connector.hadoop2.version>0.13.1-hadoop2</bigquery.connector.hadoop2.version>
```

Comment:
why do we need this change? Also the latest version is 0.13.4-hadoop2 why `0.13.1-hadoop2`?